proprietary/tokens/src/components/ams/file-input.tokens.json

```diff
@@ -21,9 +21,9 @@
         "box-shadow": { "value": "inset 0 0 0 {ams.border.width.md} {ams.color.primary-blue}" },
         "color": { "value": "{ams.color.primary-blue}" },
         "cursor": { "value": "{ams.action.activate.cursor}" },
-        "margin-inline-end": { "value": "{ams.space.inside.md}" },
-        "padding-block": { "value": "{ams.space.inside.xs}" },
-        "padding-inline": { "value": "{ams.space.inside.md}" },
+        "margin-inline-end": { "value": "{ams.space.md}" },
+        "padding-block": { "value": "{ams.space.sm}" },
+        "padding-inline": { "value": "{ams.space.md}" },
```

Comment:
This differs from our regular button styling, `padding-inline` is `lg` there. I must say I prefer the smaller spacing.

Reply:
We decided to have medium inline padding for button-like components. So this one adhered already.